css/index.css

```diff
@@ -1,3 +1,346 @@
-/* Example of @import use
-This imports the `reset.css` file */
 @import './reset.css';
+
+@media (min-width: 1024px) {
+  .unshow_li,
+  .dd_div {
+    display: block;
+  }
+
+  .main_section {
+    min-height: 70vh;
+  }
+
+  .col_l {
+    flex-basis: 25% !important;
+    max-width: 25% !important;
+  }
```

Comment:
Please remove all use of `!important`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. 

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the `!important` rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.

Reply:
Thanks for the review. I'll make the necessary adjustments and corrections.